pkg/cli/alpha/generate.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package alpha
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kubebuilder/v3/pkg/config/store"
+	"sigs.k8s.io/kubebuilder/v3/pkg/config/store/yaml"
+	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin/util"
+)
+
+const defaultOutputDir = "output-dir"
+
+// NewScaffoldCommand return a new scaffold command
+func NewScaffoldCommand() *cobra.Command {
+	var outputPath, inputPath string
+	scaffoldCmd := &cobra.Command{
+		Use:   "generate",
+		Short: "Re-scaffold an existing kuberbuilder project",
+		Long:  `Re-scaffold an existing kuberbuilder project`,
+		Run: func(cmd *cobra.Command, args []string) {
+			cwd, err := os.Getwd()
```

Comment:
Could we encapsulate this code in a func like run() since it will probably grow?

Reply:
Good suggestion! I will do that.